app/controllers/branding_groups_controller.rb

```diff
@@ -11,6 +11,15 @@ def index
     @branding_groups = BrandingGroup.all
   end
 
+  # show a list for display and linking, editable only if the user has appropriate permissions
+  def list_navigate
+    @editable_branding_groups = []
+    @branding_groups = BrandingGroup.visible_groups_to_user(current_user)
+    if current_user.present?
+      @editable_branding_groups = current_user.available_branding_groups.to_a
```

Comment:
Is there a reason we're specifically casting this to an array with `.to_a`?  The `order_by` method returns a `Mongoid::Criteria` which will respond to all the same iteration methods as an `Array` (as they are descendants of `Enumerable`).  Just curious.

Reply:
The idea was to make sure the function always returned the same 'type' of object regardless of user being signed in or not -- either an empty array, or a full array.  You're right that since they share most of the key interaction methods, it would have been ok to return the criteria also